package.json

```diff
@@ -53,6 +53,7 @@
     "debug": "^4.3.3",
     "firebase-admin": "^10.0.2",
     "firebase-functions": "^3.18.1",
+    "lodash.merge": "^4.6.2",
```

Comment:
Would do `npm uninstall lodash.merge` and then `npm install lodash -w web` 

This package.json is to be used for global packages

Reply:
I forgot about that, thank you!